front_left_wheel_joint:
  p: 100.0
  i: 0.1
  d: 10.0
front_right_wheel_joint:
  p: 100.0
  i: 0.1
  d: 10.0
rear_left_wheel_joint:
  p: 100.0
  i: 0.1
  d: 10.0
rear_right_wheel_joint:
  p: 100.0
  i: 0.1
  d: 10.0

front_left_steering_joint:
  p: 100.0
  i: 0.1
  d: 10.0
front_right_steering_joint:
  p: 100.0
  i: 0.1
  d: 10.0
rear_left_steering_joint:
  p: 100.0
  i: 0.1
  d: 10.0
rear_right_steering_joint:
  p: 100.0
  i: 0.1
  d: 10.0